doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -365,19 +400,27 @@ To enable Rivest-Shamir-Adleman (RSA) support, set one or more of the following
 RSA driver configurations
 =========================
 
-You can use the following Kconfig options for fine-grained control over which drivers provide RSA support:
+You can use the following table to check the RSA algorithm support of each driver:
+
++-----------------------+---------------------------+----------------------------+
+| RSA algorithms        | nrf_cc3xx driver support  | nrf_oberon driver support  |
++=======================+===========================+============================+
+| RSA OAEP              | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| RSA PKCS#1 v1.5 crypt | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| RSA PKCS#1 v1.5 sign  | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| RSA PSS               | Not supported             | Supported                  |
++-----------------------+---------------------------+----------------------------+
+
+The option :kconfig:option:`PSA_USE_CC3XX_SIGNATURE_DRIVER` enables the driver :ref:`nrf_security_drivers_cc3xx` for the RSA PKCS#1 v1.5 signing algorithm.
```

Comment:
```suggestion
The option :kconfig:option:`CONFIG_PSA_USE_CC3XX_SIGNATURE_DRIVER` enables the driver :ref:`nrf_security_drivers_cc3xx` for the RSA PKCS#1 v1.5 signing algorithm.
```

Ignore if this is a nit.

Reply:
(could possibly cause a CI issue)

Reply:
Interesting that the doc build didn't fail before but I will append CONFIG to all the symbols in the doc file, thanks